pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -59,18 +59,19 @@ func Validate(ic *types.InstallConfig) error {
 
 func validateMachinePool(con *ovirtsdk.Connection, child *field.Path, pool *ovirt.MachinePool) field.ErrorList {
 	allErrs := field.ErrorList{}
-	allErrs = append(allErrs, validateInstanceTypeID(con, child, pool))
+	allErrs = append(allErrs, validateInstanceTypeID(con, child, pool)...)
 	return allErrs
 }
 
-func validateInstanceTypeID(con *ovirtsdk.Connection, child *field.Path, machinePool *ovirt.MachinePool) *field.Error {
+func validateInstanceTypeID(con *ovirtsdk.Connection, child *field.Path, machinePool *ovirt.MachinePool) field.ErrorList {
+	var allErrs field.ErrorList
```

Comment:
I think this may still be `nil`? Probably better to initialize like the `validateMachinePool`: 	
```
allErrs := field.ErrorList{}
```